website/docs/en/plugins/list/plugin-react.mdx

```diff
@@ -145,3 +145,20 @@ pluginReact({
   },
 });
 ```
+
+### enableProfiler
+
+When set to true, enables the React Profiler for performance analysis in production builds. Use the React DevTools to examine profiling results and identify potential performance optimizations. Profiling adds a slight overhead, so it's disabled by default in production environments.
+
+```ts title="rsbuild.config.ts"
+pluginReact({
+  // Only enable the profiler when REACT_PROFILER is true, as the option will increase the build time and adds some small additional overhead.
+  enableProfiler: process.env.REACT_PROFILER === 'true',
+});
+```
+
+In your env file, define `REACT_PROFILER`. You can customize the environment variable name if desired.
+
+```env title=".env"
+REACT_PROFILER=true
```

Comment:
How about adding `REACT_PROFILER=true` to CLI commands instead of defining it in the env file.

```bash
REACT_PROFILER=true npx rsbuild build
```

I'm concerned that the use of .env files may lead to users applying profiles to production bundles  and release them online.

Reply:
That would be better 👍